src/bin/rav1e.rs

```diff
@@ -195,11 +193,23 @@ fn main() {
 
   if video_info.bit_depth == 8 {
     do_encode::<u8, y4m::Decoder<'_, Box<dyn Read>>>(
-      cfg, cli.verbose, progress, err, &mut cli.io.output, &mut source, y4m_enc
+      cfg,
+      cli.verbose,
+      progress,
+      err,
+      &mut *cli.io.output,
+      &mut source,
+      y4m_enc
```

Comment:
Unrelated cosmetic changes.

Reply:
it's formatted by cargo fmt.
I will change it.